covenant-signer/docs/transition-from-phase1.md

```diff
@@ -0,0 +1,378 @@
+[Babylon Node Setup Guide](https://github.com/babylonlabs-io/networks/blob/main/bbn-test-5/babylon-node/README.md).
+
+One you have the `babylond` binary installed,
+navigate to the directory you want to set up your keyring and run:
+
+```shell
+babylond keys import-hex cov fe1c56c494c730f13739c0655bf06e615409870200047fc65cdf781837cf7f06 \
+    --keyring-backend file \
+    --keyring-dir /path/to/your/keyring/directory
+```
+
+This command:
+- Uses `import-hex` to import the raw private key
+- Names the key `cov` in the keyring
+- Uses the secure `file` backend which encrypts the key on disk
+- Will prompt you for a passphrase to encrypt the key
+
+The passphrase you set here will be needed later on, keep this in mind.
+
+> ⚡ Note: While both `os` and `file` backends are supported, we recommend 
+using the `file` backend as it has been thoroughly tested across different 
+environments. The `file` backend stores the private key in encrypted form 
+on disk. When running `import-hex` with the `file` backend, you will be 
+prompted for a passphrase. This passphrase will be required to unlock the 
+signer later.
+
+To confirm that the import was successful, run:
+
+```shell
+babylond keys show cov
+```
+
+The output will display the details of the imported key:
+
+```shell
+    address: bbn1azasawj3ard0ffwj04zpxlw2pt9cp7kwjcdqmc
+    name: cov
+    pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"Ajp5tUbHnX98X/IGINkUtc9yUGMdEvbiZCftnT+Yxcyx"}'
+    type: local
+```
+
+Congratulations! You have successfully imported your key.
+
+## 4. Operation
+### 4.1. Configuration
+
+Next, we can return to the covenant signer directory 
+and create your own configuration file.
+
+Use the example configuration [file](../example/config.toml) to create your own 
+configuration file. Then, replace the placeholder values with your own 
+configuration. This can be placed directly in the `covenant-signer` directory.
+
+```toml
+[keystore]
+keystore-type = "cosmos"
+
+[keystore.cosmos]
+# pointing to the directory where the key is stored, unless specified otherwise
+key-directory = "/path/to/keydir"
+
+# the backend to be used for storing the key, in this case file
+keyring-backend = "file"
+
+# the name of the key you used when importing the key
+key-name = "your-key-name"
+
+# the chain id of the chain to be used
+chain-id = "your-chain-id"
+
+[server-config]
+# The IP address where the covenant-signer server will listen
+host = "127.0.0.1"
+# The TCP port number where the covenant-signer server will listen
+port = 9791
+
+[metrics]
+# The IP address where the Prometheus metrics server will listen
+host = "127.0.0.1"
+# This port is used to expose metrics that can be scraped by Prometheus
+port = 2113
+```
+
+Below are brief explanations of the configuration entries:
+
+- `keystore-type`: Type of keystore used. Should be set to `"cosmos"`
+- `key-directory`: Path where keys are stored on the filesystem.
```

Comment:
alert that the key-directory should be without the suffix of the keyring-backend type

like the expected is `github.com/babylonlabs-io/covenant-emulator/covenant-signer/test-bbn` instead of `github.com/babylonlabs-io/covenant-emulator/covenant-signer/test-bbn/keyring-file`